runtime/src/main/java/org/corfudb/runtime/view/StreamsView.java

```diff
@@ -69,13 +70,13 @@ public IStreamView get(UUID stream, StreamOptions options) {
      * @param streamIDs    The streams to append to.
      * @param object       The object to append to each stream.
      * @param conflictInfo Conflict information for the sequencer to check.
+     * @param cacheOption  The caching mode for write/append
      * @return The address the entry was written to.
      * @throws TransactionAbortedException If the transaction was aborted by
      *                                     the sequencer.
      */
     public long append(@Nonnull Object object, @Nullable TxResolutionInfo conflictInfo,
-                       @Nonnull UUID ... streamIDs) {
-
+                       @Nonnull CacheOption cacheOption, @Nonnull UUID ... streamIDs) {
```

Comment:
Maybe use lombok's NonNull (already imported) rather than javax's. Otherwise, lombok's import is unnecessary and can be removed

Reply:
done.